docs/content/en/blog/releases/v0.33.0.md

```diff
@@ -8,12 +8,34 @@ description: >
 
 ## Changelog since v0.32.4
 
+### Breaking Changes
+
+According to a recently discovered [vulnerability](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-24348), it reveals that the existence of a directory traversal vulnerability when an arbitrary path can be specified as the Helm values file path.
+
+For this reason, PipeCD has restricted the path that can be specified as the values file path to the directory where the application configuration (i.e. `.pipecd.yaml`) exists when a local path is specified by [#3726](https://github.com/pipe-cd/pipecd/pull/3726).
```

Comment:
```suggestion
For this reason, PipeCD has restricted the path that can be specified as the values file path to the directory where the application configuration (i.e. `app.pipecd.yaml`) exists when a local path is specified by [#3726](https://github.com/pipe-cd/pipecd/pull/3726).
```

Reply:
Oops, [this](https://github.com/pipe-cd/pipecd/pull/3744#discussion_r894129615) is fine